docs/reference/docs/bulk.asciidoc

```diff
@@ -110,10 +110,14 @@ something similar on the client side, and reduce buffering as much as
 possible.
 
 There is no "correct" number of actions to perform in a single bulk request.
-Experiment with different settings to find the optimal size for your particular workload.
-
-When using the HTTP API, make sure that the client does not send HTTP chunks,
-as this will slow things down.
```

Comment:
I am not sure I understand the slow down of this and assume it is on purpose that you removed it. But curious if you found out why this is there (I see Clinton added it in 2013).

Reply:
Yes I'm also not sure where this came from and removing it was deliberate. Sending requests in chunks should actually be a little faster because it parallelises the accumulation of items in the bulk request with the sending of the request to ES. Even if it was slower in 2013 I don't think it's true today, or at least if it is then we should fix it rather than just documenting it.